Pilot/pilotTools.py

```diff
@@ -618,8 +614,6 @@ def __initCommandLine2(self):
         self.installation = v
       elif o == '-p' or o == '--platform':
         self.platform = v
-      elif o == '-m' or o == '--maxNumberOfProcessors':
```

Comment:
Removing --maxNumberOfProcessors will break multiprocessor support in grid and VM pilots, as they both use it on the dirac-pilot command line. In the VMs its used to distinguish between running 8 processor payloads or 8 single processor payloads.